front-end/src/pages/Register/index.js

```diff
@@ -1,6 +1,33 @@
 import React from 'react';
 
-const Register = () => (
-  <p>This is the register page</p>
-);
+import {
+  Button,
+  Checkbox,
+  FormControl,
+  FormLabel,
+  // FormErrorMessage,
+  Input,
+} from '@chakra-ui/core';
+
+const Register = () => {
```

Comment:
Josias, não sei se você viu mas o github actions deu uma chiada nessa linha aqui, por um motivo super besta, mas que não sei se um CodeClimate da vida poderia te dar uma dor de cabeça aqui. Quando você for escrever um componente funcional que não tem state nenhum, você pode usar o retorno implícito do ES6 pra escrever ele. Tipo assim: 

```suggestion
const Register = () => 
  (
    <div>
    ...
   );
```